### DISCUSSION TITLE
What was the inspiration to use scheme?

### DISCUSSION THREAD

**Discussion by jaidetree** [nvim-treesitter/nvim-treesitter#3556] [General]
I love lisp languages like Clojure and Fennel personally so was really surprised to find the syntaxes defined with scheme. What inspired that choice? 

Was looking at the Clojure syntax def and that code is really quite nice and straight forward https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/clojure/highlights.scm 🚀

**Comment by stsewd** [comment]
That's what tree-sitter itself uses 

- https://tree-sitter.github.io/tree-sitter/syntax-highlighting#queries
- https://tree-sitter.github.io/tree-sitter/using-parsers#pattern-matching-with-queries

Not really sure if there is a deeper explanation why s-expressions were used.

**Reply by jaidetree** [reply]
Ah I did not catch that thanks! I'll try asking there. If I recall, S-expressions are easy\fast to build parsers for.
